JS_lesson4/Task3.js

```diff
@@ -0,0 +1,29 @@
+// Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+const age = Number(prompt(`Введи возраст`));
+console.log(age, typeof age)
+const age_2 = 18;
+const age_3 = 60;
+switch (true) {
+    case isNaN(age): {
```

Comment:
Same as for previous task, i belive that isNaN check should be performed before switch